.github/workflows/galaxy_ci.yml

```diff
@@ -6,11 +6,14 @@ on:
     paths:
       - 'roles/galaxy_post_install/**'
       - 'roles/pulp_devel/tasks/galaxy_ui.yml'
+      - '.github/workflows/galaxy_ci.yml'
+      - 'molecule/galaxy-release/**'
+      - 'molecule/galaxy-source/**'
```

Comment:
I hate to be a strickler, but these should be renamed to release-galaxy and source-galaxy.

The convention is "what you're installing from - what type of install you are doing"